.github/workflows/create-release.yml.disabled

```diff
@@ -0,0 +1,164 @@
+name: Create GitHub Release
+
+on:
+  pull_request_review:
+    types: [submitted]
+
+jobs:
+  create-release:
+    runs-on: ubuntu-latest
```

Comment:
@MarcelChludzinski-Chili  is this fine for the OSS stuff or do we also need to use an internal runner